### ISSUE TITLE
Cannot find module 'babel-plugin-transform-runtime'

### ISSUE THREAD

**Issue by Ezku** [storybooks/storybook#1876]
EDIT: Duplicate of https://github.com/storybooks/storybook/issues/1729

Hello,

I'm upgrading to webpack 3, and for that to happen I also need to upgrade `@storybook/react` from `3.1.3` to `3.2.8`. After the upgrade, storybook fails to find `babel-plugin-transform-runtime`.

```
Error: Cannot find module 'babel-plugin-transform-runtime'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.resolve (internal/module.js:27:19)
    at Object.<anonymous> (app/node_modules/@storybook/react/dist/server/config/babel.js:12:77)
```

The module seems to be missing from the dependency tree altogether.

```
storybook/react@3.2.8 node_modules/@storybook/react
├── lodash.pick@4.4.0
├── lodash.flattendeep@4.4.0
├── case-sensitive-paths-webpack-plugin@2.1.1
├── commander@2.11.0
├── util-deprecate@1.0.2
├── @storybook/addons@3.2.6
├── json5@0.5.1
├── json-stringify-safe@5.0.1
├── @storybook/addon-links@3.2.6
├── uuid@3.1.0
├── qs@6.5.1
├── serve-favicon@2.4.4 (ms@2.0.0, fresh@0.5.1, parseurl@1.3.2, etag@1.8.1, safe-buffer@5.1.1)
├── configstore@3.1.1 (xdg-basedir@3.0.0, graceful-fs@4.1.11, unique-string@1.0.0, dot-prop@4.2.0, make-dir@1.0.0, write-file-atomic@2.3.0)
├── @storybook/channel-postmessage@3.2.0 (@storybook/channels@3.2.0)
├── babel-preset-minify@0.2.0 (babel-plugin-transform-remove-debugger@6.8.5, babel-plugin-minify-infinity@0.2.0, babel-plugin-transform-minify-booleans@6.8.3, babel-plugin-transform-undefined-to-void@6.8.3, lodash.isplainobject@4.0.6, babel-plugin-transform-remove-console@6.8.5, babel-plugin-minify-replace@0.2.0, babel-plugin-minify-numeric-literals@0.2.0, babel-plugin-transform-merge-sibling-variables@6.8.6, babel-plugin-transform-simplify-comparison-operators@6.8.5, babel-plugin-transform-regexp-constructors@0.2.0, babel-plugin-transform-member-expression-literals@6.8.5, babel-plugin-transform-inline-consecutive-adds@0.2.0, babel-plugin-minify-flip-comparisons@0.2.0, babel-plugin-minify-type-constructors@0.2.0, babel-plugin-minify-guarded-expressions@0.2.0, babel-plugin-minify-constant-folding@0.2.0, babel-plugin-transform-remove-undefined@0.2.0, babel-plugin-minify-builtins@0.2.0, babel-plugin-minify-dead-code-elimination@0.2.0, babel-plugin-transform-property-literals@6.8.5, babel-plugin-minify-mangle-names@0.2.0, babel-plugin-minify-simplify@0.2.0)
├── file-loader@0.11.2 (loader-utils@1.1.0)
├── url-loader@0.5.9 (mime@1.3.6, loader-utils@1.1.0)
├── prop-types@15.5.10 (loose-envify@1.3.1)
├── webpack-hot-middleware@2.19.1 (ansi-html@0.0.7, html-entities@1.2.1, querystring@0.2.0, strip-ansi@3.0.1)
├── style-loader@0.17.0 (loader-utils@1.1.0)
├── chalk@2.1.0 (escape-string-regexp@1.0.5, supports-color@4.4.0, ansi-styles@3.2.0)
├── global@4.3.2 (process@0.5.2, min-document@2.19.0)
├── express@4.15.4 (setprototypeof@1.0.3, escape-html@1.0.3, array-flatten@1.1.1, cookie-signature@1.0.6, utils-merge@1.0.0, content-type@1.0.4, merge-descriptors@1.0.1, methods@1.1.2, encodeurl@1.0.1, vary@1.1.1, range-parser@1.2.0, fresh@0.5.0, parseurl@1.3.2, cookie@0.3.1, path-to-regexp@0.1.7, serve-static@1.12.4, etag@1.8.1, content-disposition@0.5.2, statuses@1.3.1, depd@1.1.1, qs@6.5.0, on-finished@2.3.0, finalhandler@1.0.5, proxy-addr@1.1.5, debug@2.6.8, send@0.15.4, type-is@1.6.15, accepts@1.3.4)
├── react-modal@2.3.2 (react-dom-factories@1.0.1, exenv@1.2.2)
├── glamorous@4.9.3 (html-tag-names@1.1.2, svg-tag-names@1.1.1, fast-memoize@2.2.8, is-function@1.0.1, brcast@3.0.1, is-plain-object@2.0.4, react-html-attributes@1.4.1)
├── webpack-dev-middleware@1.12.0 (time-stamp@2.0.0, path-is-absolute@1.0.1, range-parser@1.2.0, mime@1.4.0, memory-fs@0.4.1)
├── find-cache-dir@1.0.0 (commondir@1.0.1, make-dir@1.0.0, pkg-dir@2.0.0)
├── @storybook/addon-actions@3.2.6 (deep-equal@1.0.1, react-inspector@2.1.4)
├── postcss-flexbugs-fixes@3.2.0 (postcss@6.0.11)
├── common-tags@1.4.0
├── airbnb-js-shims@1.3.0 (es5-shim@4.5.9, es6-shim@0.35.3, function.prototype.name@1.0.3, string.prototype.padstart@3.0.0, promise.prototype.finally@3.0.1, object.getownpropertydescriptors@2.0.3, array-includes@3.0.3, string.prototype.padend@3.0.0, object.values@1.0.4, object.entries@1.0.4)
├── shelljs@0.7.8 (interpret@1.0.4, glob@7.1.2, rechoir@0.6.2)
├── glamor@2.20.40 (object-assign@4.1.1, through@2.3.8, inline-style-prefixer@3.0.8)
├── postcss-loader@2.0.6 (loader-utils@1.1.0, postcss-load-config@1.2.0, postcss@6.0.11, schema-utils@0.3.0)
├── request@2.82.0 (aws-sign2@0.7.0, forever-agent@0.6.1, tunnel-agent@0.6.0, oauth-sign@0.8.2, caseless@0.12.0, is-typedarray@1.0.0, safe-buffer@5.1.1, stringstream@0.0.5, aws4@1.6.0, isstream@0.1.2, extend@3.0.1, performance-now@2.1.0, combined-stream@1.0.5, mime-types@2.1.17, tough-cookie@2.3.2, form-data@2.3.1, hawk@6.0.2, http-signature@1.2.0, har-validator@5.0.3)
├── autoprefixer@7.1.4 (normalize-range@0.1.2, num2fraction@1.2.2, postcss-value-parser@3.3.0, browserslist@2.4.0, postcss@6.0.11, caniuse-lite@1.0.30000733)
├── css-loader@0.28.7 (object-assign@4.1.1, lodash.camelcase@4.3.0, postcss-value-parser@3.3.0, source-list-map@2.0.0, loader-utils@1.1.0, babel-code-frame@6.26.0, css-selector-tokenizer@0.7.0, postcss-modules-extract-imports@1.1.0, postcss-modules-scope@1.1.0, postcss-modules-local-by-default@1.2.0, icss-utils@2.1.0, postcss-modules-values@1.3.0, cssnano@3.10.0)
├── babel-preset-react@6.24.1 (babel-plugin-syntax-jsx@6.18.0, babel-plugin-transform-react-jsx-self@6.22.0, babel-preset-flow@6.23.0, babel-plugin-transform-react-jsx-source@6.22.0, babel-plugin-transform-react-display-name@6.25.0, babel-plugin-transform-react-jsx@6.24.1)
├── babel-preset-env@1.6.0 (babel-plugin-syntax-trailing-function-commas@6.22.0, babel-plugin-transform-es2015-spread@6.22.0, babel-plugin-transform-es2015-arrow-functions@6.22.0, babel-plugin-transform-es2015-template-literals@6.22.0, babel-plugin-transform-es2015-destructuring@6.23.0, babel-plugin-transform-es2015-literals@6.22.0, babel-plugin-transform-es2015-for-of@6.23.0, babel-plugin-check-es2015-constants@6.22.0, babel-plugin-transform-es2015-block-scoped-functions@6.22.0, babel-plugin-transform-es2015-typeof-symbol@6.23.0, semver@5.4.1, invariant@2.2.2, browserslist@2.4.0, babel-plugin-transform-es2015-block-scoping@6.26.0, babel-plugin-transform-es2015-modules-amd@6.24.1, babel-plugin-transform-es2015-computed-properties@6.24.1, babel-plugin-transform-es2015-modules-umd@6.24.1, babel-plugin-transform-es2015-unicode-regex@6.24.1, babel-plugin-transform-es2015-sticky-regex@6.24.1, babel-plugin-transform-es2015-modules-commonjs@6.26.0, babel-plugin-transform-es2015-duplicate-keys@6.24.1, babel-plugin-transform-es2015-shorthand-properties@6.24.1, babel-plugin-transform-es2015-parameters@6.24.1, babel-plugin-transform-es2015-classes@6.24.1, babel-plugin-transform-es2015-modules-systemjs@6.24.1, babel-plugin-transform-es2015-object-super@6.24.1, babel-plugin-transform-async-to-generator@6.24.1, babel-plugin-transform-es2015-function-name@6.24.1, babel-plugin-transform-exponentiation-operator@6.24.1)
├── babel-preset-react-app@3.0.2 (babel-plugin-transform-react-constant-elements@6.23.0, babel-plugin-transform-runtime@6.23.0, babel-plugin-transform-react-jsx-self@6.22.0, babel-plugin-transform-react-jsx-source@6.22.0, babel-plugin-transform-object-rest-spread@6.23.0, babel-plugin-transform-react-jsx@6.24.1, babel-plugin-transform-regenerator@6.24.1, babel-preset-env@1.5.2)
├── babel-preset-stage-0@6.24.1 (babel-plugin-transform-do-expressions@6.22.0, babel-plugin-transform-function-bind@6.22.0, babel-preset-stage-1@6.24.1)
├── babel-core@6.26.0 (slash@1.0.0, babel-messages@6.23.0, path-is-absolute@1.0.1, private@0.1.7, babel-helpers@6.24.1, babel-template@6.26.0, babylon@6.18.0, convert-source-map@1.5.0, source-map@0.5.7, debug@2.6.8, minimatch@3.0.4, babel-types@6.26.0, babel-code-frame@6.26.0, babel-generator@6.26.0, babel-traverse@6.26.0, lodash@4.17.4, babel-register@6.26.0)
├── babel-plugin-react-docgen@1.7.0 (babel-types@6.26.0, react-docgen@2.18.0)
└── @storybook/ui@3.2.7 (lodash.debounce@4.0.8, lodash.sortby@4.7.0, deep-equal@1.0.1, @storybook/components@3.2.7, keycode@2.1.9, events@1.1.1, @hypnosphi/fuse.js@3.0.9, @storybook/react-fuzzy@0.4.0, podda@1.2.2, react-komposer@2.0.0, react-inspector@2.1.4, mantra-core@1.7.0, react-split-pane@0.1.66, react-treebeard@2.0.3, react-icons@2.2.5)
```

Installing `babel-plugin-transform-runtime` on the app level had no effect. Neither did uninstalling and reinstalling `@storybook/react`. Any suggestions on what I could do?

```
𝚫  git:(upgrade-webpack-3) ✗ npm -v
2.15.11
𝚫  git:(upgrade-webpack-3) ✗ node -v
v6.4.0
```

Thanks,
Eevert

**Comment by valscion** [comment]
I think this is precisely #1729 that was fixed by #1848. There just hasn't been any new storybook release for 3.2.x versions

**Comment by Hypnosphi** [comment]
Now it's released as [3.2.10](https://github.com/storybooks/storybook/releases/tag/v3.2.10)

**Comment by valscion** [comment]
Thank you! 💞
